ui/app/components/app/connected-sites-list/connected-sites-list.container.js

```diff
@@ -9,19 +9,17 @@ import {
 import {
   getRenderablePermissionsDomains,
   getPermissionsDomains,
-  getAddressConnectedToCurrentTab,
   getSelectedAddress,
 } from '../../../selectors/selectors'
 import { getOriginFromUrl } from '../../../helpers/utils/util'
 
 const mapStateToProps = (state) => {
-  const addressConnectedToCurrentTab = getAddressConnectedToCurrentTab(state)
   const { openMetaMaskTabs } = state.appState
   const { title, url, id } = state.activeTab
 
   let tabToConnect
 
-  if (!addressConnectedToCurrentTab && url && !openMetaMaskTabs[id]) {
+  if (url && !openMetaMaskTabs[id]) {
```

Comment:
I think we want to keep this. This ensures that the manual connect for legacy dapps is hidden for accounts that are already connected.